web/app/login.html

```diff
@@ -25,21 +27,48 @@ <h1  class="mdc-typography--headline1">Login</h1>
         </span>
         <span class="mdc-notched-outline__trailing"></span>
       </span>
-      <input type="email" class="mdc-text-field__input" aria-labelledby="my-label-id">
-    </label><br>
-    <label class="mdc-text-field mdc-text-field--outlined">
+      <input
+        type="email"
+        class="mdc-text-field__input"
+        aria-labelledby="my-label-id"
+      /> </label
+    ><br />
```

Comment:
```suggestion
    >
```

Reply:
Please, don't use <br> cause it is bad for accessibility. You can always replace <br> with margin-bottom for label :)